### ISSUE TITLE
Websocket works incorrectly

### ISSUE THREAD

**Issue by AkioSarkiz** [swooletw/laravel-swoole#487]
1. PHP 8, swoole 4.6.6
2. Laravel 8
3. 2.8.1
4. When i send emit to concrete user, message sent to current user + my id;
5. I upload images with code and result. 
6. -

it's my route
```php
Websocket::on('example', function ($websocket, $data) {
   $websocket->toUserId([1])->emit('notification', 'only for you message id: 1');
});
```

User with login=admin & id=2, sent example emit and received notification event for user by id 1.
![img](https://i.imgur.com/79AGHG3.png)

User with login=user & id=1, too received this message.
![img](https://i.imgur.com/kjlam7k.png)

**Comment by Arkanius** [comment]
Thanks for open this issue!

I'll check it out

**Comment by AkioSarkiz** [comment]
Three months have passed.  And the bug was not fixed

**Comment by albertcht** [comment]
Hi @Arkanius ,

What is your expected result? Can't understand your question yet.

```
Websocket::on('example', function ($websocket, $data) {
   $websocket->toUserId([1])->emit('notification', 'only for you message id: 1');
});
```
> It will listen to `example` event, and always send a message to user 1 when this event is triggered.

**Comment by Arkanius** [comment]
Hi @albertcht! 

I think he's saying that the "emit" is always sending the message to the user with id 1 and 2 when it should send just to the user with id 1

**Comment by AkioSarkiz** [comment]
yes, if user with id `2` send a message to only for `1`, user with id `1` gets message, but and user with id `2` gets message

**Comment by Arkanius** [comment]
@AkioSarkiz  What about the other users? I mean, if you have user 3. This user will get the message as well?

**Comment by AkioSarkiz** [comment]
@Arkanius no, only current + 2
